routes/routes.py

```diff
@@ -8,5 +8,21 @@ def check():
 
 @router.route("/add", methods=["POST"])
 def add():
-    # Add logic here
-    return
\ No newline at end of file
+    x = request.form.data.param1
+    y = request.form.data.param2
+
+    if x > y:
+        smaller = y
+    else:
+        smaller = x
+    for i in range(1, smaller+1):
```

Comment:
Calculating the HCF under the `/add route` and sending a response with a message claiming that the numbers have been added - this does not sound meaningful, friend.

Why don't you create a separate route for computing the hcf, and just send a simple add operation here? :)